src/components/news-page/news-page.component.jsx

```diff
@@ -7,68 +7,77 @@ import newsImage3 from '../../assets/news_img_03.jpg';
-                />
-                <NewsContentGroup
-                    url="https://www.bbc.co.uk/news"
-                    image={newsImage4}
-                    article="年内営業日・年始営業開始日についてのお知らせ"
-                    date="2020.12.27"
-                    mediaType="Press Release"
-                />
-                <NewsContentGroup
-                    url="https://www.bbc.co.uk/news"
-                    image={newsImage5}
-                    article="緊急事態宣言発令に伴う、当社の対応について"
-                    date="2020.04.08"
-                    mediaType="Press Release"
-                />
-                <NewsContentGroup
-                    url="https://www.bbc.co.uk/news"
-                    image={newsImage6}
-                    article="新商品のお知らせ"
-                    date="2019.10.27"
-                    mediaType="Press Release"
-                />
+            <div className="container-container">
+                <div className="news-content-group-container">
 
+                    <NewsContentGroup
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage1}
+                        article="2021年、明けましておめでとうございます"
+                        date="2021.01.04"
+                        mediaType="Press Release"
+                    />
+                    <NewsContentGroup
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage2}
+                        article="雑誌『Organic』にて、当社商品が紹介されました！"
+                        date="2020.08.10"
+                        mediaType="Media"
+                    />
+                    <NewsContentGroup
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage3}
+                        article="DEAR COCOから新しいライフスタイルを提案"
+                        date="2020.01.15"
+                        mediaType="Press Release"
+                    />
+                    <NewsContentGroup
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage4}
+                        article="年内営業日・年始営業開始日についてのお知らせ"
+                        date="2020.12.27"
+                        mediaType="Press Release"
+                    />
+                    <NewsContentGroup
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage5}
+                        article="緊急事態宣言発令に伴う、当社の対応について"
+                        date="2020.04.08"
+                        mediaType="Press Release"
+                    />
+                    <NewsContentGroup
+                        url="https://www.bbc.co.uk/news"
+                        image={newsImage6}
+                        article="新商品のお知らせ"
+                        date="2019.10.27"
+                        mediaType="Press Release"
+                    />
+
+                </div>
+            </div>
+            <img className="news-bg-leaf" src={messageLeaf} alt="bg-leaf" />
```

Comment:
when the browser is squished, the image seems to stop the AnimatedButton from being pressed.

Reply:
fixed